### ISSUE TITLE
Header mobile menu logo accessiblity

### ISSUE THREAD

**Issue by nikolab-pn** [telekom/scale#1448]
**Scale Version**
@telekom/scale-components 3.0.0-beta.113

**Framework and version**
@telekom/scale-components-react 3.0.0-beta.113

**Current Behavior**
There are two logos with links that are output by a screen reader. The second link is only visible when viewport is smaller than 1024px and is only output when reading linearly. Because both links do the same, the second link is redundant.

**Expected Behavior**
The second link should be hidden from a screen reader by using aria-hidden=“true“if it cannot be removed.

![image](https://user-images.githubusercontent.com/98950408/210749883-18593572-7ea7-4eda-8675-5ec414e745fe.png)
